app/views/shared/_like.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_with model: @like, url: "/users/#{params[:user_id]}/posts/#{params[:id]}/likes/create" do |form| %>
+  <%= form.submit "Like"%>
+<% end %>
```

Comment:
- [ ] I cannot create a like too. Kindly fix that. You may need to ensure your route is well set up.

  ![image](https://github.com/Bellagirl-maker/blog_app/assets/54848363/9e6150a6-5894-4f04-8377-5dbf228dc244)

Reply:
@DammyShittu done with this change